br/pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -1694,13 +1671,18 @@ func (cpdb *MySQLCheckpointsDB) DestroyErrorCheckpoint(ctx context.Context, tabl
 		}
 
 		// Delete the checkpoints
-		if _, e := tx.ExecContext(c, deleteChunkQuery, tableName); e != nil {
+		if _, e := tx.ExecContext(c, `DELETE FROM ?.? WHERE table_name IN (SELECT table_name FROM ?.?
```

Comment:
table name is not a SQL parameter, we still need to use printf to build it. See https://github.com/pingcap/tidb/pull/49662